packages/sentry/src/sentry.ts

```diff
@@ -21,45 +21,44 @@ export const boot = () => {
 };
 
 export const reportErrorToSentry = (
-  error: Error | string,
+  error: Error | ErrorEvent | PromiseRejectionEvent | string,
   extraInfo?: { extra?: object | string },
   getIsEnabled?: () => boolean
 ) => {
   const isEnabled = getIsEnabled
     ? getIsEnabled()
     : Boolean(window.app.trackingSentry);
 
-  if (error instanceof Error === false && !isEnabled) {
+  if (typeof error === 'string' && !isEnabled) {
     console.warn(
-      '[SENTRY]: You called "reportErrorToSentry" with an argument that is not an instance of "Error". ' +
-        '"Error" should be preferred so that a stack-trace can be made available in sentry. ' +
+      '[SENTRY]: You called "reportErrorToSentry" with a string argument. ' +
+        '"Error" objects should be preferred so that a stack-trace can be made available in Sentry. ' +
         'See: https://docs.sentry.io/clients/javascript/usage/#try-catch'
     );
   }
 
   if (isEnabled) {
-    // logs error to sentry
-    Sentry.withScope(scope => {
```

Comment:
Since we want to return a value from the `reportErrorToSentry` function, we can't use a callback here. Luckily, Sentry exposes some methods from the static export, so we don't need to use `withScope`.